docs/user/metadata.md

```diff
@@ -60,6 +60,10 @@ writer.add_metadata(
 # Save the new PDF to a file
 with open("meta-pdf.pdf", "wb") as f:
     writer.write(f)
+
+#Access new custom field (while standard fields like Author have primitive analogs and can be accessed like reader.metadata.author, custom fields do not)
```

Comment:
Thanks for the PR. I would have thought that this is obvious, but apparently it is not?

Regarding the changes itself:

* Please keep a consistent style, id est starting comments with a whitespace, using `reader` over custom names and using double quotes.
* What are "primitive analogs"? Why not just use the term property which Python developers should know already?